workspaces/ui/src/contexts/SpecViewerContext.js

```diff
@@ -0,0 +1,57 @@
+import React, { useMemo } from 'react';
+import axios from 'axios';
```

Comment:
any reason to use axios instead of fetch?

Reply:
i was running into issues with `fetch` — but that was before we did a lot of fixing — just retried it and it all worked so I switched over

Reply:
awesome. Thank you!